<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaxWindow</class>
 <widget class="QMainWindow" name="StaxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>828</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Stax</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="StaxCalendar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ListTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>661</width>
      <height>192</height>
     </rect>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotLine</enum>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="TopMenuFileBar">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="FileActionCreateGoal"/>
    <addaction name="FileActionRemoveGoal"/>
    <addaction name="FileActionUpdateGoal"/>
   </widget>
   <widget class="QMenu" name="TopMenuHelpBar">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="HelpActionAboutStax"/>
   </widget>
   <addaction name="TopMenuFileBar"/>
   <addaction name="TopMenuHelpBar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="HelpActionAboutStax">
   <property name="text">
    <string>About Stax</string>
   </property>
  </action>
  <action name="FileActionCreateGoal">
   <property name="text">
    <string>Create Goal</string>
   </property>
  </action>
  <action name="FileActionRemoveGoal">
   <property name="text">
    <string>Remove Goal</string>
   </property>
  </action>
  <action name="FileActionUpdateGoal">
   <property name="text">
    <string>Update Goal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
